### PR TITLE
Lead feature/task1_add_preferential_tax_zone

### PR DESCRIPTION
**Pull request by LogicAdept**

### CODE REVIEW

File: src/com/company/model/Account.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.company.model;
 
+import javax.swing.plaf.synth.Region;
```
Comment (parmaj) [inline src/com/company/model/Account.java:3]: Are you sure that you need this class?

File: src/com/company/model/Account.java [hunk 2]
```diff
@@ -14,15 +15,30 @@
     protected BigDecimal sumValue;
     protected List<Payment> payments;
     protected Integer taxRate = 13;
+
```
Comment (parmaj) [inline src/com/company/model/Account.java:18]: There are lots of empty spaces

File: src/com/company/model/Account.java (hunk 1 above)
Comment (LogicAdept) [reply src/com/company/model/Account.java:3]: Oh my god, its trash

File: src/com/company/model/Account.java (hunk 2 above)
Comment (LogicAdept) [reply src/com/company/model/Account.java:18]: yea,  a lot

File: src/com/company/model/Account.java (hunk 1 above)
Comment (LogicAdept) [reply src/com/company/model/Account.java:3]: fixed

File: src/com/company/model/Account.java (hunk 2 above)
Comment (LogicAdept) [reply src/com/company/model/Account.java:18]: fixed

**Pull request merged**